linux/just_git_airgap_repo.bsh

```diff
@@ -1172,39 +1298,86 @@ function submodule-helper-relative-url()
 }
 
 #**
-# .. function:: git_sync_submodules
+# .. function:: git_submodule_sync_recursive
+#
+# ``git submodule sync --recursive`` but using a specific remote (``just_upstream`` by default)
+#
+# .. seealso::
+#
+#   :func:`git_submodule_sync`
+#**
+
+function git_submodule_sync_recursive()
+{
+  _recursive_sync=0 git_submodule_sync ${@+"${@}"}
+}
+
+#**
+# .. function:: git_submodule_sync
 #
 # ``git submodule sync`` but using a specific remote (``just_upstream`` by default)
 #
 # Like ``git submodule sync``, synchronize the configured URL of each submodule in a repository (recursively) to the value specified in .gitmodules; however, instead of overwriting the default remote (origin), update the remote specified by JUST_GIT_UPSTREAM (``just_upstream`` by default).
 #
 # :Arguments: [``$1``] - The project-repository's remote. Default: origin
+#             [``$2...``] - The submodule path to sync (or multiple submodules). Default: all submodules of the current repository (non-recursively)
 # :Parameters: - [``JUST_GIT_UPSTREAM``] - The submodules' remote to sync. When updating the URL in the submodules' configuration, use this remote. Default: ``just_upstream``
-# :Parameters: [``RECURSIVE``] - Should the submodules be iterated through recursively. Default: unset
+#
+# .. note::
+#
+#   Because this function uses ``git submodule foreach``, it skips submodules that have been init'd but not updated.
```

Comment:
Do we need to add a check to git airgap repo somewhere, to detect uninited submodules, and issue a warning?